.github/actions/create-jira-issues-for-community-activities/action.yml

```diff
@@ -100,21 +100,26 @@ runs:
       if: steps.issue-related-pr.outputs.is-issue-pr == 'False'
       shell: pwsh
       run: |
-        # Note that while --fail will fail the command if the request fails, it won't show the server output. Thus,
-        # remove for debugging. --fail-with-body provides that too, but is only available under curl >=7.76.0 which is
-        # not yet available for GitHub Actions, see https://github.com/Lombiq/GitHub-Actions/issues/53.
-        curl --request POST `
-          --url '${{ env.JIRA_BASE_URL }}/rest/api/3/issue/${{ steps.create-issue.outputs.issue }}/remotelink' `
-          --user '${{ env.JIRA_USER_EMAIL }}:${{ env.JIRA_API_TOKEN }}' `
-          --header 'Accept: application/json' `
-          --header 'Content-Type: application/json' `
-          --fail `
-          --data '{
-                \"object\": {
-                    \"url\":\"${{ steps.issue-details.outputs.link-url }}\",
-                    \"title\":\"${{ steps.issue-details.outputs.link-title }}\"
-                }
-            }'
+        $linkProperties = @{
+            object = @{
+                url = '${{ steps.issue-details.outputs.link-url }}'
+                title = '${{ steps.issue-details.outputs.link-title }} opened with Invoke-WebRequest'
```

Comment:
` opened with Invoke-WebRequest` to be removed from the title!

Reply:
Looks good!

Reply:
Reverted and deleted test issues and repo.